platform/security/sts/security-sts-wss/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+
+ -->
+<metatype:MetaData xmlns:metatype="http://www.osgi.org/xmlns/metatype/v1.2.0">
+
+    <OCD description="STS WSS Configuration Settings"
+         name="Security STS WSS"
+         id="ddf.security.sts.wss.configuration">
+
+        <AD name="SAML Assertion Type:" id="assertionType"
+            description="The version of SAML to use. Most services require SAML v2.0. Changing this value from the default could cause services to stop responding."
+            required="true" type="String"
+            default="http://docs.oasis-open.org/wss/oasis-wss-saml-token-profile-1.1#SAMLV2.0">
+            <Option label="SAML v2.0"
+                    value="http://docs.oasis-open.org/wss/oasis-wss-saml-token-profile-1.1#SAMLV2.0"/>
+            <Option label="SAML v1.1"
+                    value="http://docs.oasis-open.org/wss/oasis-wss-saml-token-profile-1.1#SAMLV1.1"/>
+        </AD>
+
+        <AD name="SAML Key Type:" id="keyType"
+            description="The key type to use with SAML. Most services require Bearer. Changing this value from the default could cause services to stop responding."
+            required="true" type="String"
+            default="http://docs.oasis-open.org/ws-sx/ws-trust/200512/Bearer">
+            <Option label="Bearer" value="http://docs.oasis-open.org/ws-sx/ws-trust/200512/Bearer"/>
+            <Option label="PublicKey"
+                    value="http://docs.oasis-open.org/ws-sx/ws-trust/200512/PublicKey"/>
+            <Option label="SymmetricKey"
+                    value="http://docs.oasis-open.org/ws-sx/ws-trust/200512/SymmetricKey"/>
+        </AD>
+
+        <AD name="SAML Key Size:" id="keySize"
+            description="The key size to use with SAML. The default key size is 256 and this is fine for most applications. Changing this value from the default could cause services to stop responding."
+            required="true" type="String"
```

Comment:
Same note, unless there's a specific reason it should be String that I'm unaware of.